Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -184,3 +184,18 @@ public struct KlaviyoSDK {
         return false
     }
 }
+
+// MARK: - SPI Private methods
+
+// for internal use only
+
+@_spi(KlaviyoPrivate)
```

Comment:
should we add it to `Sources/KlaviyoCore/Networking/PrivateMethods.swift`?

Reply:
I thought about that but ended up putting it here because that file is in the `KlaviyoCore/Networking` directory, and this method is neither within the KlaviyoCore library, nor is it networking code.

Personally I think it makes sense to keep it within Klaviyo.swift, as it's easier to find that way, but it's also clearly marked as private and it's separated out from the public API using an extension.